remotehandler.py

```
@@ -0,0 +1,294 @@
+                    'Parts': [
+                        {
+                            'PartNumber': part['PartNumber'],
+                            'ETag': part['ETag']
+                        }
+                        for part in self.multipart_upload['uploaded parts']
+                    ]
+                }
+            )
+            print(f'Upload successful. Uploaded {self.multipart_upload["index"]-1} segments.')
+            return response
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)